addons/l10n_ke/data/template/account.tax-ke.csv

```diff
@@ -1,101 +1,101 @@
-"id","description","invoice_label","type_tax_use","name","amount_type","amount","tax_group_id","l10n_ke_item_code_id","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/tag_ids","repartition_line_ids/factor_percent","repartition_line_ids/account_id"
-"ST16","Sales VAT (16%)","Sales VAT (16%)","sale","16%","percent","16.0","tax_group_16","","base","invoice","+16% Sales Base","",""

```

Comment:
It would not be bad to have it in the commit msg as well that we are changing the invoice_label